frontend/src/app/features/work-packages/components/wp-buttons/wp-status-button/wp-status-button.component.sass

```diff
@@ -6,7 +6,7 @@
     padding: 5px
     margin-bottom: 0
     background-color: var(--status-selector-bg-color)
-    border: none
+    border: 1px solid var(--button--border-color)
```

Comment:
Since it's a button and each button on Github has a border, I added a border for WP status button. If it's not intended, I can change it to have border only in high-contrast mode.

Reply:
I see.. Please double check with product about this.